homeassistant/components/sensor/ecoal_boiler.py

```diff
@@ -0,0 +1,101 @@
+"""
+Allows reading temperatures from ecoal/esterownik.pl controller.
+
+Example configuration:
+sensor:
+  - platform: ecoal_boiler
+    enable:
+      outdoor_temp: Outdoor
+      indoor_temp: Konrad room
+      indoor_temp2: Ala room
+      domestic_hot_water_temp: Domestic hot water
+      feedwater_out_temp: Feedwater
+      # more sensors from SENSOR_IDS
+
+ecoal_boiler:
+  host: 192.168.1.123
+  login: admin
+  password: admin
+
```

Comment:
This should go into the documentation.

Reply:
Docs PR created: https://github.com/home-assistant/home-assistant.io/pull/7707